SpatialGDK/Source/SpatialGDKTests/Private/RPCContainer/ObjectSpy.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Utils/RPCContainer.h"
+
+#include "CoreMinimal.h"
+
+#include "ObjectSpy.generated.h"
+
+namespace SpyUtils
+{
+	TArray<uint8> TypeToArray(ESchemaComponentType Type);
```

Comment:
these need more descriptive names

Reply:
What would you suggest?

Reply:
Something that says it's a component enum type to a byte array specifically.

Reply:
ToByteArray and ToSchemaType would probably do.

Reply:
Renamed to SchemaTypeToByteArray and ByteArrayToSchemaType.